src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,13 @@
+spring:
+  cloud:
+    propertiesvolume:
+      paths: /mnt/secrets/cmc
+      prefixed: false
+      aliases:
+        claim-store-s2s-secret: IDAM_S2S_AUTH_TOTP_SECRET
+        notify-api-key: GOV_NOTIFY_API_KEY
+        citizen-oauth-client-secret: oauth2.client.secret
+        anonymous-caseworker-username: IDAM_CASEWORKER_ANONYMOUS_USERNAME
+        anonymous-caseworker-password: IDAM_CASEWORKER_ANONYMOUS_PASSWORD
+        system-update-username: IDAM_CASEWORKER_SYSTEM_USERNAME
+        system-update-password: IDAM_CASEWORKER_SYSTEM_PASSWORD
```

Comment:
You will have to add alias mapping for passwords in here. They won't impact till you add secrets to mount .

Reply:
I wonder if spring will automatically relax bind claim-store-db-password to CLAIM_STORE_DB_PASSWORD . it might do it to claimStoreDbPassword.

Reply:
this works though 🤔 green build with smoke/functional tests passing - so spring is binding db password (and any other secrets that match) ok. what do you think? does something still need doing? :)

Reply:
It might be working because you are not taking passwords from secrets on preview- they are coming from values.aat.teamplte.yaml.